solutions/dedicated/README.md

```diff
@@ -0,0 +1,11 @@
+# IBM Cloudant Dedicated on IBM Cloud Dedicated
+
+This architecture creates an IBM Cloudant Dedicated instance in IBM Cloud Dedicated and provisions the following resources:
```

Comment:
I think this needs to be re-worded:
`provisions the following resources` -> `supports provisioning the following resources` otherwise it seems as if the DA will create a database by default, which is not the case (its an opt in feature)